UnityProject/AddressablePackingProjects/SoundAndMusic/Packages/packages-lock.json

```diff
@@ -61,7 +61,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.scriptablebuildpipeline": {
-      "version": "1.15.2",
```

Comment:
Hmm, this wasn't available in 2021.1.14f1. Probably don't want this for now?

Reply:
ok, yeah, the forced Serialisation changes drowned a lot of stuff